src/api/__tests__/chat.test.ts

```diff
@@ -1,15 +1,15 @@
 import { chatWithAI } from '../chat';
-import { callSidecar } from '../sidecar';
+import { callSidecarWithArgs } from '../sidecar-with-args';
 
-vi.mock('../sidecar');
```

Comment:
`vi.mock('module')` mocks all the functions in the module. We only want it to mock one (`callSidecarWithArgs`, not `callSidecar`). The easiest way is to mock just one function in a module is to just move it into a standalone module. https://stackoverflow.com/q/59312671/388951

Reply:
(this is irrelevant after the refactor to use Pydantic)